src/openrct2/drawing/sprite.cpp

```diff
@@ -34,8 +34,8 @@ extern "C"
 extern "C"
 {
     static void *   _g1Buffer = nullptr;
-    static rct_gx   _g2;
-    static rct_gx   _csg;
+    static rct_gx   _g2 = { 0 };
+    static rct_gx   _csg = { 0 };
```

Comment:
This is redundant since they are static which is guaranteed to be zero init by the standard unless otherwise specified or is a constant. Also in C++ `= {}` works as well.